workers/unity/Packages/com.improbable.gdk.tools/GdkToolsConfiguration.cs

```diff
@@ -89,6 +96,7 @@ private static class DefaultValues
             public const string CodegenOutputDir = "Assets/Generated/Source";
             public const string SchemaSourceDir = "../../schema";
             public const string RuntimeIp = null;
+            public const string DevAuthTokenDir = "Resources";
```

Comment:
because the DAT generator prepends Application.dataPath to this (which atm points to the root `Assets/` dir)